workflows/nf-tractoflow.nf

```diff
@@ -89,6 +94,68 @@ workflow NF_TRACTOFLOW {
                 .map{ it + [[]] }
         )
 
+    if (params.run_atlas_based_tractometry) {
+        ATLAS_IIT()
+        ch_versions = ch_versions.mix(ATLAS_IIT.out.versions)
+
+        // Register IIT atlas to subject space
+        ch_input_register_iit = TRACTOFLOW.out.b0
+            .combine(ATLAS_IIT.out.b0)
+            .map{ meta, b0, template_b0 -> [meta, b0, template_b0, []] }
+        REGISTER_ATLAS_B0(ch_input_register_iit)
+
+        // Apply the transformation to subject space to the bundles
+        ch_iit_transform_bundles = TRACTOFLOW.out.b0
+            .join(REGISTER_ATLAS_B0.out.forward_image_transform)
+            .combine(ATLAS_IIT.out.bundle_masks.toList())
+            .map {
+                meta, b0, transform, bundles ->
+                    [meta, bundles, b0, transform]
+            }
+        TRANSFORM_ATLAS_BUNDLES(ch_iit_transform_bundles)
+        ch_versions = ch_versions.mix(TRANSFORM_ATLAS_BUNDLES.out.versions)
+
+        // Prepare volume ROI metric extraction
+        // Start by collecting DTI metrics
+        ch_input_metricsinroi = TRACTOFLOW.out.dti_fa
+            .join(TRACTOFLOW.out.dti_md)
+            .join(TRACTOFLOW.out.dti_rd)
+            .join(TRACTOFLOW.out.dti_ad)
```

Comment:
I would highly suggest you to grab as many metrics available as possible (NODDI, FREEWATER, AFD...) if we run them it will use them.